contrib/cookiecutter/ckan_extension/hooks/post_gen_project.py

```diff
@@ -36,15 +36,22 @@ def recut():
     setup_template = 'setup.py'
 
     # get context
-    context = {{ cookiecutter | jsonify }}
+    context = {}
+
+{%- for key, value in cookiecutter.items() -%}
+{% if value is not none %}
+    context["{{key}}"] = {{ value|tojson }}
+{% else %}
+    context["{{key}}"] = None
+{% endif %}
```

Comment:
This might cover the edge cases better

```suggestion
    context["{{key}}"] = {{ value.__repr__() | safe }}
```